cypress/e2e/marc/marc-bibliographic/create-new-marc-bib/automated-linking/auto-linking-button-disabled.cy.js

```diff
@@ -0,0 +1,168 @@
+    { rowIndex: 14, tag: '651', content: '$aAfrica.$2fast $0 sh 85001531' },
+    { rowIndex: 15, tag: '655', content: '$aComics (Graphic works)$2fast $0 gf2014026266' },
+    {
+      rowIndex: 16,
+      tag: '700',
+      content: '$aLee, Stan,$d1922-2018,$ecreator.$0 http://id.loc.gov/authorities/names/n83169267',
+    },
+    { rowIndex: 17, tag: '710', content: '$aRobinson $0 no2008081921' },
+    { rowIndex: 18, tag: '711', content: '$aDelaware $0 n  84745425' },
+    { rowIndex: 19, tag: '730', content: '$aGone T $0n 79066095' },
+    { rowIndex: 20, tag: '800', content: '$aNeilson, Donald $0 n 79023811' },
+    { rowIndex: 21, tag: '810', content: '$aBlack Panther Party $0 n 80095585' },
+    {
+      rowIndex: 22,
+      tag: '811',
+      content: '$aStockholm International Film Festival $0 no2018125587',
+    },
+    { rowIndex: 23, tag: '830', content: '$aBlack Panther $0 no2018018754' },
+  ];
+
+  let userData = {};
+
+  const linkableFields = [
+    100, 110, 111, 130, 240, 600, 610, 611, 630, 650, 651, 655, 700, 710, 711, 730, 800, 810, 811,
+    830,
+  ];
+
+  const marcFiles = [
+    {
+      marc: 'marcAuthFileForC389484.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 20,
+    },
+  ];
+
+  const createdAuthorityIDs = [];
+  let createdInstanceID;
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorCreate.gui,
+      Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
+    ]).then((createdUserProperties) => {
+      userData = createdUserProperties;
+
+      cy.loginAsAdmin().then(() => {
+        marcFiles.forEach((marcFile) => {
+          cy.visit(TopMenu.dataImportPath);
+          DataImport.verifyUploadState();
+          DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+          JobProfiles.waitLoadingList();
+          JobProfiles.search(marcFile.jobProfileToRun);
+          JobProfiles.runImportFile();
+          JobProfiles.waitFileIsImported(marcFile.fileName);
+          Logs.checkStatusOfJobProfile('Completed');
+          Logs.openFileDetails(marcFile.fileName);
+          for (let i = 0; i < marcFile.numOfRecords; i++) {
+            Logs.getCreatedItemsID(i).then((link) => {
+              createdAuthorityIDs.push(link.split('/')[5]);
+            });
+          }
+        });
+      });
+
+      linkableFields.forEach((tag) => {
+        QuickMarcEditor.setRulesForField(tag, false);
+      });
+
+      cy.login(userData.username, userData.password, {
+        path: TopMenu.inventoryPath,
+        waiter: InventoryInstances.waitContentLoading,
+      });
+    });
+  });
+
+  after('Deleting created users, Instances', () => {
```

Comment:
Could you please reset auto-linking rules after the test to default values, or just set them to `true` for all fields?